src/app/modules/consultations/consultation-profile/consultation-profile.graphql.ts

```diff
@@ -96,6 +96,22 @@ export const ConsultationProfile = gql`
       updatedAt
       publishedAt
       visibility
+      consultationPartnerResponses {
```

Comment:
Move this to a fragment and reuse it in the queries

Reply:
@gpalsingh - Bro, after creating a fragment and using it in the query we are getting an error. So moved it to previous state

Reply:
@Bandinikhil - What is the error and why cannot it be fixed?

Reply:
@gpalsingh - there was an typename error for fragment, now it;s fixed bro.